Lib/test/test_builtin.py

```diff
@@ -1511,6 +1513,126 @@ def test_construct_singletons(self):
             self.assertRaises(TypeError, tp, 1, 2)
             self.assertRaises(TypeError, tp, a=1, b=2)
 
+
+class TestBreakpoint(unittest.TestCase):
+    def test_breakpoint(self):
+        with patch('pdb.set_trace') as mock:
```

Comment:
Hum, would it be possible to modify the unit test so it doesn't rely on breakpoint() calling pdb.set_trace()? The test may be run with PYTHONBREAKPOINT set to something else.. to debug the test (or another test while running the test suite). Or skip the test if sys.breakpointhook is no sys.__breakpointhook__?